packages/astro/package.json

```
@@ -166,7 +166,7 @@
     "resolve": "^1.22.4",
     "semver": "^7.5.4",
     "server-destroy": "^1.0.1",
-    "shikiji": "^0.6.13",
+    "shikiji": "^0.9.17",
```

```suggestion
    "shikiji": "^0.9.18",
```
Can we upgrade the library version to the suggested version to support Angular v17? Please refer to this changelogs https://github.com/antfu/shikiji/compare/v0.9.17...v0.9.18